DTAdmin/Groups/FacultyForGroupViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FacultyViewController.swift
+//  DTAdmin
+//
+//  Created by Admin on 21.10.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class FacultyForGroupViewController: UIViewController {
+    
+    @IBOutlet weak var facultyTableView: UITableView!
+    
+    var selectFaculty: ((Faculty) -> ())?
+    var faculties = [Faculty]() {
```

Comment:
Please write SwiftDoc to describe that table view is reloaded when a user/client set the `faculties` property